### DISCUSSION TITLE
mspider/user-guide/service-list/diagnose

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2921] [Announcements]
# mspider/user-guide/service-list/diagnose

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/user-guide/service-list/diagnose.html

**Comment by nicole-lihui** [comment]
添加诊断部分与服务状态区分的说明

区分必要与不必要？与服务接入规范相结合
